bindings/python/slimt.cpp

```diff
@@ -77,7 +77,9 @@ class PyService {
     for (auto &handle : handles) {
       auto &future = handle.future();
       future.wait();
-      responses.push_back(std::move(future.get()));
+      Response response = future.get();
+      change_ranges_to_utf8(response);
+      responses.push_back(std::move(response));
```

Comment:
The following should allow the calling client to control whether to transform indices or not. This should allow you to get the response without the transformation, in python use `text.encode()[range.begin:range.end]` to compare with the one with transformation in python (`text[range.begin:range.end]`), and craft a test without hard-coding values.

What do you think? You might have to pybind the new enum, though. 

```diff
 std::vector<Response> translate(std::shared_ptr<Model> model, py::list &texts,
-                                  bool html) {
+                                  bool html, 
+                                  Encoding encoding = Encoding::UTF8) {
   ...
   
+    if(encoding == Encoding::UTF8){
+         change_ranges_to_utf8(response);
+    }
}
```